src/plans/stopping_criterion.jl

```diff
@@ -364,6 +464,25 @@ function (c::StopWhenAll)(p::AbstractManoptProblem, s::AbstractManoptSolverState
     end
     return false
 end
+function status_summary(c::StopWhenAll)
+    has_stopped = length(c.reason) > 0
+    s = has_stopped ? "reached" : "not reached"
+    r = "Stop When _all_ of the following are fulfilled:\n"
+    for cs in c.criteria
+        r = "$r    $(status_summary(cs))\n"
+    end
+    return "$(r)Overall: $s"
+end
+function indicates_convergence(c::StopWhenAll)
+    return any(indicates_convergence(ci) for ci in c.criteria)
```

Comment:
Shouldn't it be `all`?

Reply:
I thought so first as well. But it is any.
If this indicates to stop, then _all_ its containing ones indicate that. If only one of them indicated true, then it did converge.
For the StopWhenAny it is also not any but one has to check whether the active ones (the ones that indicated stop) have one of them who indicated stop.

So Any should be fine.

Reply:
I see, that makes sense.